packages/node_modules/@webex/plugin-meetings/UPGRADING.md

```diff
@@ -0,0 +1,286 @@
+# Meetings Plugin Usage Guide
+
+The `plugin-phone` plugin is deprecated and replaced with `plugin-meetings`.
+With this change comes significant code changes in upgrading to the meetings pluing.
+The meetings plugin is much more feature rich and maintained than the phone plugin.
+
+For users of the phone plugin, one of the main changes from `plugin-phone` is the idea of having a `meeting` object that we are performing actions on.
+Instead of just "calling" a destination, a meeting object must be "created" and then "joined".
+
+## Setup
+
+### Is Calling Supported
+
+The phone plugin provided a `isCallingSupported()` method to detect if WebRTC was supported in the user's browser.
+
+This method is no longer provided and if the functionality is needed, we suggest you use the [DetectRTC Package on NPM](https://www.npmjs.com/package/detectrtc).
+
+### Device Registration
+
+Meeting data is received via websockets from the Webex Teams servers.
+In order for the client to receive these websocket events, it must register with the websocket servers.
+
+When using the phone plugin, this was handled automatically for you via `Webex.phone.register()`.
+
+With the meetings plugin, device registration and websocket connection need to be manually handled:
+
+```js
+webex.internal.device.register()
+  .then(() => webex.internal.mercury.connect());
+```
+
+## Dialing
+
+Previously, users of the phone plugin could do a `webex.phone.dial(destination)` command.
+Unfortunately, this "single line of code" method is not possible with the features that the meetings plugin provides.
+Instead, a meeting object must be created, joined and media added.
+This new method is much more deliberate and less "magical", allowing the developer to fully control all steps of the calling process.
+
+## Meeting Creation
+
+As stated above, before any actions can be performed on a meeting, the object must be created. This is done via:
+
+```js
+const myMeeting = await webex.meetings.create(destination);
+```
+
+## Joining a Meeting
+
+Once a meeting object has been created, to start it, simply `join` it:
+
+```js
+await myMeeting.join();
+
+// Now, Add Media
+```
```

Comment:
Is there an example for adding media? Probably one of the most confusing parts of the plugin

Reply:
Yes, just below these samples is a add media section